charset_normalizer/md.py

```diff
@@ -153,9 +137,8 @@ def eligible(self, character: str) -> bool:
 
     def feed(self, character: str) -> None:
         if (
-            character not in {"\n", "\t", "\r", "\v"}
+            character.isspace() is False  # includes \n \t \r \v
             and character.isprintable() is False
-            and character.isspace() is False
             and ord(character) != 0x1A  # Why? Its the ASCII substitute character.
```

Comment:
Is there a reason to use `ord()` here?

```
$ python -m timeit 'ord(" ") != 0x1A'
5000000 loops, best of 5: 47.4 nsec per loop
$ python -m timeit '" " != "\x1A"'
10000000 loops, best of 5: 24.3 nsec per loop
```

Reply:
It does seem better! @Ousret what do you think?

Reply:
Even faster: `" " is not "\x1A"`

Reply:
Might be faster, but incorrect. You can not use `is not` comparison safely here.